chapter_recurrent-neural-networks/rnn_vn.md

```diff
@@ -362,7 +365,10 @@ Let us look at a number of cases:
 4. What are some of the problems associated with the simple sequence model described above?
 -->
 
-*dịch đoạn phía trên*
+1. Nếu sử dụng RNN để dự đoán ký tự tiếp theo trong chuỗi văn bản thì ta sẽ cần đầu ra có bao nhiêu chiều?
+2. Thử thiết kế một ánh xạ trong đó một RNN với các trạng thái ẩn  chính xác. Gợi ý: một số lượng từ hữu hạn thì sao? <!-- Mình chưa hiểu rõ câu hỏi này, bạn nào review gợi ý giúp mình nhé. Thanks -->
```

Comment:
```suggestion
2. Thử thiết kế một ánh xạ trong đó một các trạng thái ẩn của RNN là chính xác (không chỉ là xấp xỉ). Gợi ý: một số lượng từ hữu hạn thì sao?
```
hidden states của Rnn sẽ chứa thông tin chuỗi mà nó đã nhìn thấy, nhưng mà số lượng nút trong hidden states thường nhỏ nên không thể chứa toàn bộ nội dung mà phải tóm tắt, xấp xỉ chứ ko còn chính xác nữa ý.
Để có hidden state chính xác thay vì xấp xỉ thì số lượng nút trong hidden states lớn bằng số lượng vocab, lúc đó hidden state có thể là biểu diễn one hot chẳng hạn.

Reply:
Giờ mình hiểu rồi. Cám ơn @lkhphuc  nhé!